cmd/nerdctl/login.go

```diff
@@ -82,6 +82,11 @@ func processLoginOptions(cmd *cobra.Command) (types.LoginCommandOptions, error)
 		password = strings.TrimSuffix(string(contents), "\n")
 		password = strings.TrimSuffix(password, "\r")
 	}
+
+	if strings.Contains(username, ":") {
```

Comment:
I suggest moving this cond to the ligne 64; no need to check passwd if user is invalid.

Reply:
:heavy_check_mark: done